runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -101,13 +101,13 @@ public T open() {
         try {
```

Comment:
If both the name and uuid are specified in the builder, then we need to check that the uuid can be derived from the name.

Reply:
Sounds good